### ISSUE TITLE
Use markdown for images (not R/knitr)

### ISSUE THREAD

**Issue by jules32** [Openscapes/website-new#66]
Hi @samanthacsik @stefaniebutland , I'm writing this here for us to implement going forward. 

Here's an example of markdown (green) vs R (red) syntax for images: 

<img width="714" alt="image" src="https://github.com/Openscapes/website-new/assets/5891909/2b96f4cd-d93b-462b-bf1f-8e95d6b748c4">

I know Sam had good reasons for using R for images rather than markdown, but going forward let's use markdown. The reason is portability - we'll be cross-posting on the NASA-Openscapes blog post for now and not requiring R in post creation and GitHub Action is important (these websites otherwise don't rely on code, only quarto-flavored markdown). 

See https://github.com/Openscapes/how_we_work/issues/292 for screenshots and details. 

This to me would mean using Markdown syntax going forward, and updating the wiki, but I wanted to get your thoughts?

**Comment by samanthacsik** [comment]
I agree with everything you've written here @jules32 ! That was an oversight on my part, but let's definitely move forward with the language-agnostic markdown syntax. 

I made a first pass at updates to the wiki: https://github.com/Openscapes/website-new/wiki/Adding-images

I didn't fully remove the information on `knitr::include_graphics()` since it still appears throughout the website, but open to feedback

**Comment by jules32** [comment]
This sounds great, thank you Sam for updating the wiki!

**Comment by stefaniebutland** [comment]
@samanthacsik Just used your updated wiki info for images. Thank you!! (cause sometimes you never know if ppl read your documentation)
